remoteworkitem/trackerquery_repository_blackbox_test.go

```diff
@@ -30,19 +31,13 @@ func (s *trackerQueryRepoBlackBoxTest) SetupTest() {
 
 func (s *trackerQueryRepoBlackBoxTest) TestFailDeleteZeroID() {
 	// Create at least 1 item to avoid RowsEffectedCheck
-	tr, err := s.trRepo.Create(
-		s.Ctx,
-		"http://api.github.com",
-		remoteworkitem.ProviderGithub)
-	if err != nil {
-		s.T().Error("Could not create tracker", err)
-	}
-
-	_, err = s.repo.Create(
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Trackers(1), tf.Spaces(1))
+	trackerID := fmt.Sprintf("%s", fxt.Trackers[0].ID)
```

Comment:
Please use [`strconv.Itoa`](https://golang.org/pkg/strconv/#Itoa) instead. Also in other locations below.

Reply:
I fixed this here 42c3698